core/extensions/AppWithPlugin/FrenchPlugin/FrenchPlugin.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>net5</TargetFramework>
```

Comment:
Would it make sense to add a `Directory.Build.props` to set `TargetFramework` for all these projects?

Reply:
@elinor-fung That's an interesting idea. We're looking at how to update all the samples on the release cadence.  See dotnet/docs#22271.  Can you add thoughts on how to make that easier on that issue?